qa/workunits/mon/caps.py

```diff
@@ -85,7 +86,7 @@ def __init__(self, cid, j):
     self.args = []
     for s in j['sig']:
       if not isinstance(s, dict):
-        assert isinstance(s, str) or isinstance(s,unicode), \
+        assert isinstance(s, str) or isinstance(s,six.text_type), \
```

Comment:
space after ','

Reply:
as long as we don't have automated tests for that, I don't plan to fix theses nitpicks...